### DISCUSSION TITLE
Multi-Language Functionality

### DISCUSSION THREAD

**Discussion by stanimir93** [medusajs/medusa#3295] [Feature Requests]
Can we have a multi-language functionality? 

For example, the same product could have descriptions in different languages based on the user's preference. 

It will be helpful for those who run medusa in multilingual regions/countries.

Thank you!

**Comment by olivermrbl** [comment]
Thanks! This feature has already been requested, and is therefore a duplicate of #2003. Please move any comments, ideas, or feedback there to centralise the discussion.
